_includes/hero.html

```diff
@@ -1,12 +1,16 @@
-<div class="hero row">
-	<div class="sideHighlight column">
-		<h1 class="shout">
-			<a href="/process/design.html" class="anchor-bland" action="Find out more about the design phase of our projects">Design,</a>
-			<a href="/process/build.html" class="anchor-bland" action="Find out more about the build phase of our projects">Build</a>
-			&amp; <a href="/process/grow.html" class="anchor-bland" action="Find out more about the grow phase of our projects">Grow</a>
-		</h1>
-		<p>
-			Generating measurable value through innovative design and development.
-		</p>
+<div class="hero">
+	<div class="row">
```

Comment:
Foundation allows you to do `class="row column"` so that you don't have the extra markup. Then you can get rid of the direct child div